contrib/sles11/netdata-alarms-bash3.patch

```diff
@@ -1,9 +1,6 @@
 --- system/netdata.conf.orig    2018-05-10 19:44:49.000000000 +0300
 +++ system/netdata.conf 2018-05-10 19:45:14.000000000 +0300
 @@ -22,3 +22,7 @@
- [web]
-     web files owner = root
-     web files group = netdata
 +
 +[health]
 +    # script for sles 11, mail notifications only
```

Comment:
That is likely wrong and I need properly to regenerate the patch @Ferroin, correct? If so I just undo the commit 😄

Reply:
SLES 11 is EOL. Maybe we just remove the patches, @Ferroin?